package.json

```diff
@@ -27,7 +27,7 @@
     "@mdx-js/react": "^1.6.22",
     "@nevermined-io/catalog": "^1.2.0",
     "@nevermined-io/providers": "^1.2.0",
-    "@nevermined-io/sdk": "^1.2.0",
+    "@nevermined-io/sdk": "^2.0.0",
```

Comment:
this is the root package.json for the entire documentation. any reason why we are doing this?